src/main/java/org/jabref/logic/formatter/bibtexfields/RemoveTabsFormatter.java

```diff
@@ -0,0 +1,42 @@
+package org.jabref.logic.formatter.bibtexfields;
+
+import java.util.Objects;
+import java.util.regex.Pattern;
+
+import org.jabref.logic.l10n.Localization;
+import org.jabref.model.cleanup.Formatter;
+
+/**
+ * Replaces any tab with a space
+ */
+public class RemoveTabsFormatter extends Formatter {
+
+    private static final Pattern TAB = Pattern.compile("\t+");
+
+    @Override
+    public String getName() {
+        return Localization.lang("Remove tabs");
+    }
+
+    @Override
+    public String getKey() {
+        return "remove_tabs";
+    }
+
+    @Override
+    public String format(String value) {
+        Objects.requireNonNull(value);
+
+        return TAB.matcher(value).replaceAll(" ");
```

Comment:
Oh, tabs are replaced by a space.

Maybe change the formatter name to `ReplaceTabsBySpaceFormater` and change the descriptoin accordingly.